electrum/mnemonic.py

```diff
@@ -187,38 +187,43 @@ def mnemonic_decode(self, seed):
             i = i*n + k
         return i
 
+    def random_bytes(self, num):
+        from time import time_ns
+        from secrets import token_bytes
+        hasher = hashlib.sha3_512(os.getrandom(num))
+        hasher.update(time_ns().to_bytes(16, 'big'))
+        hasher.update(token_bytes(num))
+        assert num <= hasher.digest_size
+        return hasher.digest()[:num]
+
+    def entropy_to_seed(self, entropy, wordlist):
```

Comment:
Use `self.wordlist`.
```suggestion
    def entropy_to_seed(self, entropy):
```